source/PlayerInfo.cpp

```diff
@@ -1074,6 +1074,7 @@ CargoHold *PlayerInfo::Storage(bool forceCreate)
 
 
 // Get planetary storage information for all planets (for map and overviews).
+// TODO: should CargoHold be a pointer/reference?
```

Comment:
I feel the answer to this question is no, since we are getting a reference to a map returned and the CargoHolds per planet are actually stored inside the referenced map. (And if the question is answered with no, then please remove the TODO or add some text to explain why this question could get a yes as answer.)

Reply:
Right, so there will be no extra copying, which is what I was initially concerned about. However, would using a non-pointer non-reference value type make it more difficult to change the implementation of PlayerInfo later on? (My C++ is rusty, sorry.) Also, if I'm not mistaken, all current callers need only a `const` CargoHold, not a mutable one.

Anyway, I'll just remove the comment anyway, it's not part of the PR, strictly speaking.

Reply:
I thought there might be extra copying _into_ the map; I missed that the returned value is a referenced to the actual underlying storage.

About `const`, I'll leave it for another PR.

Reply:
I just removed the comment in Rewritten in 7b7459b3054894466f79ee91752efacf359c5074..